src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/CommPkg.cs

```diff
@@ -55,6 +56,9 @@ protected CommPkg()
             Guid = guid;
         }
 
+        [NotMapped] // EF defines a shadow property for InvitationId.
```

Comment:
I don't like this .... this will cause misunderstanding since the very same class will be used in normal EF context, and then this property will never have value from DB. I think we must discuss other possible solutions with team. I stop review here for now

Reply:
I have an idea on other solution